server/routes/red-flags.js

```diff
@@ -1,10 +1,14 @@
 import express from 'express';
-import redFlags from '../controllers/redFlags.js';
+import redFlags from '../controllers/redFlags';
+
 
 const router = express.Router();
 
 router.get('/red-flags', redFlags.getAllRedFlags);
 
 router.get('/red-flags/:id', redFlags.getSingleRedFlag);
 
+router.post('/red-flags', redFlags.createRedFlag);
```

Comment:
Create a middleware to handle validations for creating a red flag. Currently, your app can be broken because you don't have validations.